share/eds/byzantine/bad_encoding_test.go

```diff
@@ -90,3 +173,39 @@ func TestIncorrectBadEncodingFraudProof(t *testing.T) {
 	err = proof.Validate(h)
 	require.Error(t, err)
 }
+
+func TestBEFP_ValidateOutOfOrderShares(t *testing.T) {
+	// skipping it for now because `malicious` package has a small issue: Constructor does not apply
+	// passed options, so it's not possible to store shares and thus get proofs for them.
+	// should be ok once app team will fix it.
```

Comment:
Please link the issue or PR. At least in the PR description

Reply:
Will create an issue once this PR is merged(to point the exact line of the code in main)